### PR TITLE
Order by

### PR DESCRIPTION
**Pull request by elliotchance**

### CODE REVIEW

File: tesseract/sql/clause/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__author__ = 'elliot'
```
Comment (elliotchance) [inline tesseract/sql/clause/__init__.py]: Remove this.

File: tesseract/sql/clause/order_by.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        assert ascending is None or isinstance(ascending, bool)
+
+        self.field_name = field_name
+        self.ascending = ascending
+
+    def __str__(self):
+        direction = ''
+        if self.ascending == True:
+            direction = ' ASC'
+        elif self.ascending == False:
```
Comment (elliotchance) [inline tesseract/sql/clause/order_by.py]: This can be just be `else`

File: tests/sql/select/order.yml [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    result:
+    - {"foo": null, "bar": 1}
+    - {"foo": "zzz", "bar": 6}
+    - {"foo": "abc", "bar": 3}
+    - {"foo": "50", "bar": 4}
+    - {"foo": 100, "bar": 5}
+    - {"foo": 50, "bar": 10}
+    - {"foo": 40, "bar": 2}
+
+  ## Edge Cases
```
Comment (elliotchance) [inline tests/sql/select/order.yml]: Fix these up.
